### PR TITLE
Added few points, fixed typo

### PR DESCRIPTION
**Pull request by bhvk121**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,12 +7,28 @@ Giving unclear requirements is worst thing that literally slow down to project p
...
 
 # 4. Bad project manager
 
+# 5. Lack of sound product owner
+
+# 6. Resource dependency 
+
+# 7. Lack of effective project management
+
+# 8. Project complexity under-estimation
```
Comment (ZaidSA) [inline README.md]: Thanks for contribution, can we make it more simpler please?

File: README.md [hunk 2]
```diff
@@ -7,12 +7,28 @@ Giving unclear requirements is worst thing that literally slow down to project p
...
+
+# 9. Frequent changes in scope
+
+# 10. Improper code merging practice
+
+# 11. Improper resource utilization / improper tasks allocation
```
Comment (ZaidSA) [inline README.md]: Should it be 2 different points?

File: README.md [hunk 3]
```diff
@@ -7,12 +7,28 @@ Giving unclear requirements is worst thing that literally slow down to project p
...
+
+# 12. unforeseen technical challenges
```
Comment (ZaidSA) [inline README.md]: unforeseen should be Unforeseen

**Review by ZaidSA — CHANGES_REQUESTED** [review]
Thanks for contribution, please check comments.

**Comment by bhvk121** [comment]
Addressed.

**Review by ZaidSA — APPROVED** [review]

**Pull request merged**